pom.xml

```diff
@@ -62,25 +62,17 @@ THE SOFTWARE.
   </developers>
 
   <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-	<java.level>8</java.level>
-	<jenkins-test-harness.version>2.13</jenkins-test-harness.version>
-	<hpi-plugin.version>1.121</hpi-plugin.version>
-    <findbugs.failOnError>false</findbugs.failOnError>
+    <jenkins.version>2.138.4</jenkins.version>
```

Comment:
Minimum Jenkins version supported by the plugin. This version is the lowest supported LTS and the recommended minimum version.